tasks/policy.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: "Cohesity Protection Policy: Set {{ cohesity_policy.name | default('policy_name') }} to state of {{ cohesity_policy.state | default('present') }}"
+  cohesity_policy:
+    cluster: "{{ cohesity_server }}"
+    username: "{{ cohesity_admin }}"
+    password: "{{ cohesity_password }}"
+    state: "{{ cohesity_policy.state | default('present') }}"
+    name: "{{ cohesity_policy.name | default('') }}"
+    description: "{{ cohesity_policy.description | default('') }}"
+    days_to_retain: "{{ cohesity_policy.days_to_retain | default(90) }}"
+    retries: "{{ cohesity_policy.retries | default(3) }}"
+    retry_interval: "{{ cohesity_policy.retry_interval | default(30) }}"
+    incremental_backup_schedule: "{{ cohesity_policy.incremental_backup_schedule }}"
+    full_backup_schedule: "{{ cohesity_policy.full_backup_schedule | default('') }}"
+    blackout_window: "{{ cohesity_policy.blackout_window | default('') }}"
```

Comment:
Finish readme along with inventory.ini task, playbook for this. giving all examples. The dicts can be confusing. Or on second thoughts check if they can just pass in the sdk model.

Reply:
I have the playbooks with module, task examples, will upload them to examples. 
I am planing for next ansible release  with clone vm's, policy, VE cluster creation and license modules. Will update the documentation for all these modules at once and make a release. 
There is no way we can pass the sdk model in the playbook